variables.tf

```diff
@@ -85,3 +85,27 @@ variable "allow_from_sgs" {
   description = "a list of security groups for which ingress rules are created"
   default     = []
 }
+
```

Comment:
@doanda86 Can you please add descriptions for these variables and make sure that you run `terraform-docs` (it's part of the pre-commit webhook, so if you install [pre-commit](https://pre-commit.com/) and set it up in this repo it will run automatically on commit)